logic/broadcaster.js

```diff
@@ -140,6 +138,11 @@ Broadcaster.prototype.enqueue = function (params, options) {
  * @return {setImmediateCallback} err | peers
  */
 Broadcaster.prototype.broadcast = function (params, options, cb) {
+	options.data.peer = library.logic.peers.me();
+	if (!options.data.peer) {
```

Comment:
good catch, the check is not needed anymore. Previously I was waiting for my own ip address being returned from a first peer connected.